### PR TITLE
Update gems friendly_id (5.1.0) and globalize (< 5.2) and fix factory_gi...

### PR DESCRIPTION
**Pull request by bricesanchez**
...rl

### CODE REVIEW

**Comment by parndt** [comment]
Thanks for your PR but CI is broken

**Comment by bricesanchez** [comment]
Haha, it's what i said to the another PR.

**Comment by parndt** [comment]
@bricesanchez yep, I pretty much quoted you :grin:

File: .travis.yml [hunk 1]
```diff
@@ -2,11 +2,8 @@ language: ruby
 sudo: false
 cache: bundler
 bundler_args: --without development
-before_script:
-  - "sh -e /etc/init.d/xvfb start"
-  - "bundle exec rake refinery:testing:dummy_app"
-script:
-  - "DISPLAY=:99.0 bundle exec rspec spec"
+before_script: "bundle exec rake refinery:testing:dummy_app"
```
Comment (parndt) [inline .travis.yml]: this only works if we're using poltergeist too

**Comment by parndt** [comment]
#418

**Pull request closed without merging**